data/en/mbs_0202.json

```diff
@@ -0,0 +1,478 @@
+                        "preview_content": {
+                            "id": "preview",
+                            "title": "Information you need",
+                            "content": [{
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                            }],
+                            "questions": [{
+                                    "question": "Total turnover",
+                                    "content": [{
+                                            "description": "Include:",
+                                            "list": [
+                                                "exports",
+                                                "income from sub-contracted activities",
+                                                "commission",
+                                                "sales of goods purchased for resale",
+                                                "transport, insurance and packaging charges",
+                                                "revenue earned from other parts of the business not named at the top of this page (please supply at fair value)"
+                                            ]
+                                        },
+                                        {
+                                            "description": "Exclude:",
+                                            "list": [
+                                                "VAT",
+                                                "income from the sales of fixed capital assets",
+                                                "grants and subsidies",
+                                                "insurance claims",
+                                                "interest received"
+                                            ]
+                                        }
+                                    ]
+                                },
+                                {
+                                    "question": "Changes in turnover",
+                                    "content": [{
+                                        "description": "Include:",
+                                        "list": [
+                                            "change in level of business activity",
+                                            "maintenance/shutdowns",
+                                            "special/calendar events",
+                                            "weather",
+                                            "price effects",
+                                            "currency effects (increase/decrease in the currency value)"
+                                        ]
+                                    }]
+                                },
+                                {
+                                    "question": "Value of exports",
```

Comment:
Think value of exports should come before, its that way in spec and makes sense from a questionnaire point of view

Reply:
The slides contradict themselves on this. I think you're right but we need to check this with @charlotte-hirst